### PR TITLE
Messages set up

### PR DESCRIPTION
**Pull request by stevenjames-turing**
## Neccesary checkmarks:

    [x] All Tests are Passing in all environments

## Description of Change:

    description: Added tables for messaging and conversations. I've started working through the process to add the websockets connection using ActionCable. We are going to have some implementing to do on the FE before this can be used, but this is a great start
    
## Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

### CODE REVIEW

**Review by pleonar1 — APPROVED** [review]
looks good.  I could follow it all and know what was going on

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 Rails.application.routes.draw do
   mount Sidekiq::Web => '/sidekiq'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  mount ActionCable.server => "/cable"
```
Comment (aedanjames) [inline config/routes.rb:5]: For my own knowlege, is "/cable" the open channel or 'endpoint address' that FE will establish a connection with?

**Comment by aedanjames** [comment]
This looks awesome, excellent setup

**Review by aedanjames — APPROVED** [review]

File: config/routes.rb (hunk 1 above)
Comment (stevenjames-turing) [reply config/routes.rb:5]: That's a solid question and I honestly don't fully know the answer. I followed a tutorial for the set-up and tried working my way through how we'll need to change the set-up for our unique needs. 

It's my understanding that we won't be creating a specific endpoint for the API with that route, but it's the the way that we can utilize the built-in ActionCable functionality to establish the open connection. On the FE, they'll hook into that route using a WebSocket specific protocol.

**Pull request merged**